prac_06/guitars.py

```diff
@@ -0,0 +1,35 @@
+from prac_06.guitar import Guitar
+
+def main():
+
+    print("My guitars!")
+    guitars = []
+    guitars.append(Guitar("Gibson L-5 CES", 1992, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.90))
+    guitars.append(Guitar("Fender Stratocaster", 2014, 765.40))
+
```

Comment:
The guitars.append information should not be there if it is used to test the program